src/daemon/ca/certauth.rs

```diff
@@ -207,15 +207,15 @@ impl Aggregate for CertAuth {
                 updates,
             } => {
                 let rc = self.resources.get_mut(&resource_class_name).unwrap();
-                let (issued, removed) = updates.unpack();
+                let (issued, removed, suspended) = updates.unpack();
                 for iss in issued {
                     rc.certificate_issued(iss)
                 }
                 for rem in removed {
                     rc.key_revoked(&rem);
 
                     // This loop is inefficient, but certificate revocations are not that common, so it's
-                    // not a big deal. Tracking this better would require that track the child handle somehow.
+                    // not a big deal. Tracking this better would require to track the child handle somehow.
```

Comment:
"to track" feels strange to me, perhaps s/that track/that we track/ or s/that track/tracking/ instead?